### DISCUSSION TITLE
Código

### DISCUSSION THREAD

**Discussion by Luadotcom** [dgadelha/Portugol-Webstudio#325] [Dúvidas / Perguntas] [answered]
Boa tarde, queria saber o que está errado nesse código. Desde já agradeço a atenção.

    inteiro idade

    escreva("Digite sua idade:")
    leia(idade)

    Se idade >= 18 Entao
      escreva("Maior de idade")
    Senao
      escreva("Menor de idade")
    FimSe

**Comment by joaojp16w** [comment]
posso estar errado, mas creio que faltou definir o valor da idade

**Reply by Luadotcom** [reply]
Obrigada, eu tentei no visualg e ele rodou certinho, já no portugol web ele dá problema. Creio que seja algo portugol web mesmo.

**Reply by dgadelha** [reply, marked as answer]
O Portugol Webstudio executa códigos Portugol apenas no padrão UNIVALI, não suportando Visualg ou outros formatos.

O código correto seria:

```portugol
programa { 
  funcao inicio() {
    inteiro idade

    escreva("Digite sua idade:")
    leia(idade)

    se (idade >= 18) {
      escreva("Maior de idade")
    } senao {
      escreva("Menor de idade")
    }
  } 
}
```
